src/lib.rs

```diff
@@ -1,3 +1,4 @@
-pub fn segfault() {
+pub fn segfault() -> ! {
     unsafe { std::ptr::null_mut::<i32>().write(1) }
+    unreachable!();
```

Comment:
this crate takes 🚀 performance 🚀 very seriously, so `std::hint::unreachable_unchecked()` should be used instead to greatly 🚀 skyrocket 🚀 performance 🚀